google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -417,4 +456,8 @@ private Logging getLogging() {
     }
     return logging;
   }
+
+  private static boolean isTrueOrNull(Boolean b) {
```

Comment:
nit: Such methods ideally should accept additional variable with default value to be returned in case if checked parameter is null.

Reply:
The method is called to answer the question whether the parameter is `null` or `Boolean.TRUE`.

Reply:
I was kinda suggesting to make it "getter", so it can be reused later if needed

Reply:
There is "getter": https://github.com/googleapis/java-logging/blob/6bc345c6e9ad0d4b3ea5dc51daf8274d8ee09c3e/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java#L401-L403

The above is a service method used at initialization only to enforce opt-out if one of the original sources of the configuration: LoggingOptions or config file has it set to FALSE. However, if the configuration is omitted then it should be interpreted as "allowed" according to the design requirement: to allow by default.